numeric-keyboard/src/main/java/com/konaire/numerickeyboard/NumericKeyboard.kt

```diff
@@ -174,5 +199,15 @@ class NumericKeyboard: FrameLayout {
 
             false
         }
+
+        keySpecial.text = keySpecialText
+        if (keySpecialText != "") {
+            keySpecial.isEnabled = true
+        }
+        if (keySpecialListener != null) {
+            keySpecial.setOnClickListener(keySpecialListener)
```

Comment:
You could simplify it like this:
```suggestion
            keySpecial.setOnClickListener(keySpecialListener ?: listener)
```